extensions/ql-vscode/src/vscode-tests/minimal-workspace/databases/db-panel.test.ts

```diff
@@ -531,6 +531,66 @@ describe("db panel", () => {
     }
   });
 
+  it("should add a new list to the remote db list", async () => {
+    const dbConfig: DbConfig = {
+      databases: {
+        remote: {
+          repositoryLists: [
+            {
+              name: "my-list-1",
+              repositories: ["owner1/repo1", "owner1/repo2"],
+            },
+          ],
+          owners: [],
+          repositories: [],
+        },
+        local: {
+          lists: [],
+          databases: [],
+        },
+      },
+      expanded: [],
+      selected: {
+        kind: SelectedDbItemKind.RemoteUserDefinedList,
+        listName: "my-list-1",
+      },
+    };
+
+    await saveDbConfig(dbConfig);
+
+    const dbTreeItems = await dbTreeDataProvider.getChildren();
+
+    expect(dbTreeItems).toBeTruthy();
+    const items = dbTreeItems!;
+
+    const remoteRootNode = items[0];
+    const remoteUserDefinedLists = remoteRootNode.children.filter(
+      (c) => c.dbItem?.kind === DbItemKind.RemoteUserDefinedList,
```

Comment:
I think you can use `isRemoteUserDefinedListDbItem` here instead of having to check the `kind`

Reply:
That could work! It's still a bit messy, because `isRemoteUserDefinedListDbItem` needs `dbItem` to not be undefined. I've left this as-is for now (since the rest of the tests also check the `kind`).